tests/kit/utils/page-logic.ts

```diff
@@ -217,3 +217,11 @@ export const addDatabaseRow = async (page: Page, databaseTitle: string) => {
   });
   await db.locator('.data-view-table-group-add-row-button').click();
 };
+
+
+export const addKanban = async (page: Page, title?: string) => {
+  await page.keyboard.press('/');
+  await expect(page.locator('affine-slash-menu .slash-menu')).toBeVisible();
+  await page.keyboard.type('kanban');
+  await page.getByTestId('Kanban View').click();
```

Comment:
I am struggling to create the equivalent test in @affine-test/affine-mobile.

On https://affine.fail on mobile (firefox), there is a menu that appears above the keyboard, but it doesn't appear for me on chrome with touch emulation enabled. This is also missing locally in chrome on `yarn run -T affine dev -p @affine/mobile`. Because this is missing and there no slash menu, I can't work out how to create a kanban view (there is no / menu on mobile).

Should I try to debug why this is and fix it, or should we merge this PR without a mobile test?